.ncurc.major.cjs

```diff
@@ -0,0 +1,5 @@
+const minorConfig = require("./.ncurc.minor.cjs");
+
+module.exports = {
+  reject: [...minorConfig.reject, "prettier"],
```

Comment:
I think `prettier` here is a typo

Reply:
Yup, both all `.ncu* files should have had empty arrays. I should check before I copy them from another repo 🙈